base/applications/rapps/include/winmain.h

```diff
@@ -15,6 +16,9 @@ struct SETTINGS_INFO
     INT Top;
     INT Width;
     INT Height;
+    /* Software source settings */
+    BOOL bUseSource;
+    WCHAR szSourceURL[INTERNET_MAX_URL_LENGTH];
```

Comment:
Adding data in the middle of the settings does break all settings below it when loading it from the registry,
And the bUseSource will probably be 0 (depending on what `Proxy` was before.
However I doubt anyone will load rapps on a machine that previously ran the old version of rapps...

Reply:
Ah, I thought ROS and RAPPS are in alpha state, do not need to think about updating ...
I just tried to make the order here correspond with the order in the dialog

Reply:
It is used by almost everyone using ReactOS, however it is almost always in a fresh install.
Either way it would make sense to store the settings in a human readable format (one value per option) some day, which also make it easier to introduce new options without having to worry about random things changing.

Reply:
almost everyone always uses rapps in a fresh install, that's why I think we can ignore this problem (at least now) 😂
one value per option does sound good. I can improve it later (probably another PR, not this one)